dev/scripts/docs-to-google-sheets.js

```diff
@@ -31,11 +35,12 @@ async function accessSpreadsheet() {
     // Create a document object using the ID of the spreadsheet - obtained from its URL.
     const doc = new googleSpreadsheet(SPREADSHEET_ID);
     // Authenticate with the Google Spreadsheets API.
-    await promisify(doc.useServiceAccountAuth.bind(doc), creds);
+    await promisify(doc.useServiceAccountAuth.bind(doc))(creds);
```

Comment:
We certainly don’t have a rule regarding calling the returned function immediately, but it doesn’t look very good :)